src/main/java/org/minions/devfund/dennis/Movie.java

```diff
@@ -0,0 +1,131 @@
+package org.minions.devfund.dennis;
+
+import java.util.ArrayList;
+
+/**
+ * This is the methods for Movie class.
+ */
+public class Movie implements Comparable<Movie> {
+    private String name;
+    private ArrayList<Actor> actors;
+    private double rating;
+
+    /**
+     * Constructor for Movie.
+     * @param name of Movie
+     */
+    public Movie(final String name) {
+        this.name = name;
+        this.rating = 0.0;
+        actors = new ArrayList<>();
+    }
+
+    /**
+     * Default values for Constructor.
+     */
+    public Movie() {
+        this("");
+    }
+
+    /**
+     * Get name of Movie.
+     * @return name of movie
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Set the name of the movie.
+     * @param name of movie
+     */
+    public void setName(final String name) {
+        this.name = name;
+    }
+
+    /**
+     * Get rating of movie.
+     * @return rating of movie
+     */
+    public double getRating() {
+        return rating;
+    }
+
+    /**
+     * Set the rating of movie.
+     * @param rating of the movie
+     */
+    public void setRating(final double rating) {
+        this.rating = rating;
+    }
+
+    /**
+     * Get the all Actors for the movie.
+     * @return the actors
+     */
+    public ArrayList<Actor> getActors() {
+        return actors;
+    }
+
+    /**
+     * Set the actors for the movies.
+     * @param actors in the array
+     */
+    public void setActors(final ArrayList<Actor> actors) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "actors" object should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)